### DISCUSSION TITLE
Feature Request: add exit magic for suggestion

### DISCUSSION THREAD

**Discussion by litingyes** [ueberdosis/tiptap#5573] [Feature Requests]
### Description

It is hoped that the **exit** logic is also triggered when the length of `props.items` is `0`

https://github.com/ueberdosis/tiptap/blob/cf2067906f506486c6613f872be8b1fd318526c9/packages/suggestion/src/suggestion.ts#L256

### Use Case

When no corresponding item is received, it is expected to hide the suggestion

### Type

New feature

**Comment by nperez0111** [comment]
I've heard this request a few times now. 
I think it is reasonable. I'd be happy to accept a PR with that change

**Reply by litingyes** [reply]
OK, I'll try.

**Reply by litingyes** [reply]
done: https://github.com/ueberdosis/tiptap/pull/5574
